packages/cli/src/util/paths.ts

```diff
@@ -0,0 +1,179 @@
+import { generateHttpParam } from '@stoplight/prism-http';
+import { serializeWithDeepObjectStyle } from '@stoplight/prism-http';
+import { serializeWithPipeDelimitedStyle, serializeWithSpaceDelimitedStyle } from '@stoplight/prism-http';
```

Comment:
You can group all these imports together.

Reply:
👍